lib/active_node/validations.rb

```diff
@@ -34,6 +34,18 @@ module Validations
     autoload :UniquenessValidator, 'active_node/validations/uniqueness'
 
     module ClassMethods
+      # Initializes and saves an object in a single step.
+      def create(attributes = nil, &block)
+        if attributes.is_a?(Array)
+          attributes.collect { |attr| create(attr, &block) }
+        else
+          object = new(attributes)
+          yield(object) if block_given?
+          object.save
+          object
+        end
+      end
+
```

Comment:
Basically yes ([see here](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/persistence.rb#L29-L54)). I did not bother to check before, but I figured that in spite of the similarities, trying to make it in a DRYer manner, would have involved some pretty nasty and not-at-all-clear meta-programming playing with the names of the methods.